compiler/pipes/init-cpp-dest-dir.cpp

```diff
@@ -0,0 +1,24 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2020 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include <atomic>
+
+#include "compiler/pipes/init-cpp-dest-dir.h"
+
+#include "compiler/compiler-core.h"
+
+void InitCppDestDir::execute(FunctionPtr function, DataStream<FunctionPtr> &os) {
+  os << function;
+
+  static std::atomic_flag dest_dir_initiated = ATOMIC_FLAG_INIT;
+  if (!dest_dir_initiated.test_and_set()) {
```

Comment:
Can you add a comment describing the purpose? (to make in understandable for a random reader)